java/ql/src/experimental/Security/CWE/CWE-078/CommandInjectionRuntimeExec.md

```diff
@@ -0,0 +1,30 @@
+# Command Injection into Runtime.exec() with dangerous command
```

Comment:
I don't think the markdowns should be uploaded to the repo, they're generated from the `.qhelp` files.

Reply:
That's a hangover from putting this in our own `codeql-queries` repo, which doesn't get that treatment from Code Scanning